Makefile

```diff
@@ -12,10 +12,10 @@ all: update-tools clean proto bins check test
 clean: clean-bins clean-test-results
 
 # Recompile proto files.
-proto: clean-proto install-proto-submodule protoc fix-proto-path proto-mock goimports-proto
+proto: clean-proto install-proto-submodule buf api-linter protoc fix-proto-path proto-mock goimports-proto
```

Comment:
This looks like a huge PR but actually only `Makefile` and content of `proto` dir worth reviewing. All the rest is just mechanical replaces.